webextensions/_locales/ru/messages.json

```diff
@@ -423,7 +421,7 @@
   "config_externalAddonPermissions_header_incognito": { "message": "Уведомлять из приватных окон" },
 
 
-  "config_debug_caption": { "message": "Разработка" },
+  "config_debug_caption": { "message": "Разработка },
```

Comment:
This is a syntax error (missing close double quotation) and I've fixed it after merged. Please merge my changes to your branch, before you update this file again.

Reply:
Ok. I got it.